passes/cmds/chformal.cc

```diff
@@ -278,21 +352,68 @@ struct ChformalPass : public Pass {
 			if (mode =='p')
 			{
 				for (auto cell : constr_cells)
-					module->addCover(NEW_ID_SUFFIX("coverenable"),
-						cell->getPort(ID::EN), State::S1, cell->get_src_attribute());
+				{
+					if (cell->type == ID($check)) {
+						Cell *cover = module->addCell(NEW_ID_SUFFIX("coverenable"), ID($check));
+						cover->attributes = cell->attributes;
+						cover->parameters = cell->parameters;
+						for (auto conn : cell->connections())
+							cover->setPort(conn.first, conn.first == ID::A ? State::S1 : conn.second);
```

Comment:
This is a minor thing, but if we do `chformal -lower; chformal -coverenable` we get the enable signal of the assertion on the `A` input of the created `$cover`, but if we do `chformal -coverenable; chformal -lower` we get it on `EN` of `$cover`

Reply:
I'll change it to be consistent